### PR TITLE
Add Lithuanian translation

### PR DESCRIPTION
**Pull request by rimas-kudelis**

### CODE REVIEW

File: i18n/lt.yaml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+- id: meta_translations
+  translation: "Vertimai"
+
+# Table of Contents
+- id: toc_title
+  translation: "Tinklalapio turinys"
+
+# Post nav
+- id: post_nav_prev
+  translation: "Ankstenis"
```
Comment (mxmehl) [inline i18n/lt.yaml:24]: Could that be a typo?

```suggestion
# Post nav
- id: post_nav_prev
  translation: "Ankstesnis"
```

**Review by mxmehl — COMMENTED** [review]
Thank you! I ran this through a tool and it gave me the following hint.

**Comment by rimas-kudelis** [comment]
Oh yes, the tool is correct!

I've committed the fix in a separate commit because I don't have this repo checked out locally, but I think it would be best to just squash these commits into one and commit using the first one's commit message.

**Comment by mxmehl** [comment]
Great, thanks for the quick fix. Merged :)

**Pull request merged**